docs/pages/database-access/guides/aws-discovery.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: AWS Database Auto-Discovery
+description: How to configure Teleport to discover AWS-hosted databases.
+---
+
+(!docs/pages/includes/database-access/discovery-guide-brief.mdx cloud="AWS" !)
+
+<Admonition type="tip">
```

Comment:
I would incorporate this information into the Prerequisites, since otherwise it strays from the focus of this guide a bit.

Maybe we could recommend either one host for both the Discovery Service and the Database Service or separate hosts for each.

Reply:
I will just remove the tip. I think the current `Prerequisites` implies a separate host for discovery service anyway, which should be the preferred configuration.